content/02.main-text.md

```diff
@@ -114,6 +114,10 @@ As of {{creation_date_pretty}}, the Deep Review repository accumulated {{total_c
 The notebook to generate this figure can be [interactively launched using Binder](https://mybinder.org/v2/gh/greenelab/meta-review/binder?filepath=analyses/deep-review-contrib/02.contrib-viz.ipynb) [@doi:10.25080/Majora-4af1f417-011], enabling users to explore alternative visualizations or analyses of the source data.
 ](images/deep-review-contribution-ridge.svg){#fig:contrib width="100%" .white}
 
+GitHub issues and even pull requests can also be used for formal peer review by independent or journal-selected reviewers.
+A reviewer conducting open peer review can create issues using their own GitHub account.
```

Comment:
Should we mention https://github.com/greenelab/meta-review/issues/124 as an example in the manuscript here.

I think this paragraph is probably the best location to mention that often repos can include peer review comments, like we do in `response-to-reviewers` to create immediately open peer review for journals that do not support it.